bin/build-site.js

```diff
@@ -73,8 +73,15 @@ if (!process.env.BUILD) {
   const http_server = require('http-server');
   const watchGlob = require('glob-watcher');
 
-  watchGlob('**', buildJekyll);
+  watchGlob('!(_site)/**', buildJekyll);
```

Comment:
This should already be handled by:

https://github.com/pouchdb/pouchdb/blob/8f388436e1d0d2f9e4d7bb18c82a9d54864b42d6/bin/build-site.js#L33-L36

It's probably worth understanding why that isn't good enough.

Reply:
...because the arg passed to glob watchers is not the path of the changed file